pkg/state/manager.go

```diff
@@ -83,7 +83,7 @@ func (smgr *stateManager) SyncState(customResource interface{}, infoCatalog Info
 	// Sync groups of states, transition from one group to the other when a group finishes
 	log.V(consts.LogLevelInfo).Info("Syncing system state")
 	managerResult := Results{
-		Status: SyncStateNotReady,
+		Status: SyncStateReady,
```

Comment:
I prefer to init it with SyncStateNotReady and in line 99 check if  done change to SyncStateReady

Reply:
Ok, I’ll rollback this change. It was done to remove one extra “if statement” and/or assignment but you’re right, let’s keep code more readable